cmd/edv-rest/startcmd/start_test.go

```diff
@@ -34,35 +35,16 @@ func TestStartCmdContents(t *testing.T) {
 	checkFlagPropertiesCorrect(t, startCmd, hostURLFlagName, hostURLFlagShorthand, hostURLFlagUsage)
 }
 
-func TestStartCmdWithBlankHostArg(t *testing.T) {
-	startCmd := GetStartCmd(&mockServer{})
-
-	args := []string{"--" + hostURLFlagName, "", "--" + databaseTypeFlagName, "mem"}
-	startCmd.SetArgs(args)
-
-	err := startCmd.Execute()
-
-	require.Equal(t, errMissingHostURL.Error(), err.Error())
-}
-
 func TestStartCmdWithMissingHostArg(t *testing.T) {
 	startCmd := GetStartCmd(&mockServer{})
 
 	err := startCmd.Execute()
 
 	require.Equal(t,
-		"neither host-url (command line flag) nor EDV_HOST_URL (environment variable) have been set",
+		"Neither host-url (command line flag) nor EDV_HOST_URL (environment variable) have been set.",
```

Comment:
your english is getting out of hand....

(yes - I haven't capitalized my sentences properly!)

Reply:
haha. The reason for this change is because before this PR, the EDV project had its own implementation of the "cmdutils" stuff, which was actually moved to edge-core at some point. I needed to use a method in the edge-core version that I didn't have here, so I just migrated it over... and it seems the version in edge-core had better English :P